airflow/docker-compose.yml

```diff
@@ -122,6 +122,9 @@ services:
             FLASK_ENV: $FLASK_ENV
             OPENEO_PYTHON_UDF_URL: $OPENEO_PYTHON_UDF_URL
             OPENEO_R_UDF_URL: $OPENEO_R_UDF_URL
+            OEO_CSW_SERVER: $OEO_CSW_SERVER
```

Comment:
I would add a comment why we now need the csw server in airflow. I assume it is connected to having the bindings as a dependency?

Reply:
yes, I added a comment